eloquent-mutators.md

```diff
@@ -449,6 +449,85 @@ protected function casts(): array
 }
 ```
 
+If you desire the collection items to be mapped into an specific class instance, you may use a second parameter or the `map()` method if you want to use the base Collection class.
+
+```php
+use App\ValueObjects\Option;
```

Comment:
According to #10186, we prefer having PHP opening tags in the code sample blocks:
```suggestion
```php
<?php

use App\ValueObjects\Option;
```

Reply:
Opening tags should go on full-code examples, I believe.

Reply:
Yeah, it is rather superfluous, but one can one do when Taylor decides that way 🤷🏻 😂